container_files/mldb_nginx_site.conf

```diff
@@ -87,12 +87,20 @@ server {
             return 301 $scheme://$http_host{{HTTP_BASE_URL}}/doc/;
         }
 
-        location {{HTTP_BASE_URL}}/doc/ {
-            # send documentation render requests to mldb (.md.html)
-            rewrite ^{{HTTP_BASE_URL}}(/?.*) $1 break;
-            try_files $uri $uri/ @mldb;
+        location = {{HTTP_BASE_URL}}/resources {
+            return 301 $scheme://$http_host{{HTTP_BASE_URL}}/resources/;
         }
 
+        # Documentation routes
+        location ~ {{HTTP_BASE_URL}}/(doc|resources)/ {
```

Comment:
We need to add a `auth_request off;` at least for the `resources` route, otherwise the licensing page breaks.